styleguide/derek/examples/_office365.html

```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+
+  <!-- Overview - Features -->
+   <div class="container-fluid feature-container bg-crumpled-paper river">
+
+      <div class="row">
+        <div class="col-md-12 center-block">
```

Comment:
Dont need center-block on col-md-12 because it spans the full-width of the row already.